Week1/Solution/2. Sum of digit

```diff
@@ -0,0 +1,6 @@
+#write a python script to enter number and sum of its digit.
+number=input('Enter Number: ')
```

Comment:
what should be happen if user input string like "vtcbb"? make changes in your program

Reply:
If the user will be enter 'vtcbb' there is an error bolecause in for loop the formula is use:

s+int(i)
So the string value will be not allowed and their sum will not possible

Reply:
Ex. Like if we input 123. It is currently string format but when the loop will be executed then 1,2,3 are converted in int using int(i).